openverse_catalog/dags/providers/provider_csv_load_scripts/inaturalist/create_schema.sql

```diff
@@ -13,25 +13,138 @@ https://github.com/inaturalist/inaturalist/blob/d338ba76d82af83d8ad0107563015364
+    notho varchar(13),
+    uninomial varchar(50),
+    genericName varchar(50),
+    infragenericEpithet varchar(25),
+    specificEpithet varchar(50),
+    infraspecificEpithet varchar(50),
+    cultivarEpithet varchar(50),
+    namePhrase varchar(80),
+    nameReferenceID varchar(36),
+    publishedInYear decimal,
+    publishedInPage varchar(255),
+    publishedInPageLink varchar(255),
+    code varchar(10),
+    nameStatus varchar(15),
+    accordingToID varchar(36),
+    accordingToPage decimal,
+    accordingToPageLink decimal,
+    referenceID text,
+    scrutinizer varchar(149),
+    scrutinizerID decimal,
+    scrutinizerDate varchar(10),
+    extinct boolean,
+    temporalRangeStart varchar(15),
+    temporalRangeEnd varchar(15),
+    environment varchar(38),
+    species decimal,
+    section decimal,
+    subgenus decimal,
+    genus decimal,
+    subtribe decimal,
+    tribe decimal,
+    subfamily decimal,
+    taxon_family decimal,
+    superfamily decimal,
+    suborder decimal,
+    taxon_order decimal,
+    subclass decimal,
+    taxon_class decimal,
+    subphylum decimal,
+    phylum decimal,
+    kingdom decimal,
+    sequenceIndex decimal,
+    branchLength decimal,
+    link varchar(240),
+    nameRemarks decimal,
+    remarks text
+);
+COMMIT;
+
+DROP TABLE IF EXISTS inaturalist.manual_name_additions;
+COMMIT;
+
+CREATE TABLE inaturalist.manual_name_additions (
+    taxon_id int,
+    vernacular_name varchar(100)
+);
+INSERT INTO inaturalist.manual_name_additions
+    (taxon_id, vernacular_name)
     VALUES
-    ('CC-BY-NC-SA', 'Creative Commons Attribution-NonCommercial-ShareAlike License', 'http://creativecommons.org/licenses/by-nc-sa/4.0/', 'by-nc-sa', '4.0'),
-    ('CC-BY-NC', 'Creative Commons Attribution-NonCommercial License', 'http://creativecommons.org/licenses/by-nc/4.0/', 'by-nc', '4.0'),
-    ('CC-BY-NC-ND', 'Creative Commons Attribution-NonCommercial-NoDerivs License', 'http://creativecommons.org/licenses/by-nc-nd/4.0/', 'by-nc-nd', '4.0'),
-    ('CC-BY', 'Creative Commons Attribution License', 'http://creativecommons.org/licenses/by/4.0/', 'by', '4.0'),
-    ('CC-BY-SA', 'Creative Commons Attribution-ShareAlike License', 'http://creativecommons.org/licenses/by-sa/4.0/', 'by-sa', '4.0'),
-    ('CC-BY-ND', 'Creative Commons Attribution-NoDerivs License', 'http://creativecommons.org/licenses/by-nd/4.0/', 'by-nd', '4.0'),
-    ('PD', 'Public domain', 'http://en.wikipedia.org/wiki/Public_domain', 'pdm', ''),
-    ('GFDL', 'GNU Free Documentation License', 'http://www.gnu.org/copyleft/fdl.html', 'gfdl', ''),
-    ('CC0', 'Creative Commons CC0 Universal Public Domain Dedication', 'http://creativecommons.org/publicdomain/zero/1.0/', 'cc0', '1.0');
+    (1, 'Animals'),
+    (47118, 'Spider'),
+    (47124, 'Flowers'),
+    (47126, 'Plants'),
+    (47157, 'Butterflies and Moths'),
+    (47158, 'Insect'),
+    (47167, 'Mushroom'),
+    (47434, 'Grass'),
+    (47604, 'Daisy'),
+    (48662, 'Monarch Butterfly');
```

Comment:
Is there any concern these `taxon_ids` could change? Would it be better to map the Latin to the English here and then link that to IDs via a join?

Reply:
That is so much a better idea!!!